qemu/tests/qemu_guest_agent.py

```diff
@@ -2409,7 +2409,7 @@ def gagent_check_fsfreeze(self, test, params, env):
 
         self._fsfreeze()
         # clean env at the end of testing
-        session.cmd(self.params["delete_temp_file"])
+        session.cmd(params["delete_temp_file"])
```

Comment:
This is not that necessary, right?

Reply:
yes, I just found it doesn't matter to add 'self', edited it easily,